cpp/oneapi/dal/algo/basic_statistics/test/online_spmd.cpp

```diff
@@ -103,8 +104,9 @@ class basic_statistics_online_spmd_test
                 }
                 partial_results.push_back(partial_result);
             }
-            const auto compute_result = this->finalize_compute_override(bs_desc, partial_results);
-
+            auto compute_result = this->finalize_compute_override(bs_desc, partial_results);
+            base_t::check_compute_result(compute_mode, data, table{}, compute_result);
+            compute_result = this->finalize_compute_override(bs_desc, partial_results);
```

Comment:
I guess its better to add a separate test with partial_fit finalize_fit partial_fit and finalize_fit, instead of checking result of twice called finalize_fit

Reply:
it would require significant code size and test runtime increase, current test change allows to check if finalize works correctly.  i'll add such scenario in intelex tests.